### PR TITLE
Document unattended installation of SAP products, including reference to SUSE Manager

### PR DESCRIPTION
**Pull request by dmacvicar**
This PR provides:

* Mention SUSE Manager in the unattended installation of the OS section, pointing to the right chapters
* Adds a section about installing SAP products with automated installation
* Provides example snippets for the SAP product section
* Makes reference to a full HANA example now included in the sap-installation wizard package.

### CODE REVIEW

**Comment by varkoly** [comment]
I've detected yesterday that the hana installation mode can be used for the TREX installation to. The only difference is that the sapMDC need not be set.
There is one more xml tab which can be used: sapVirtHostname. This becomes only important by distributed or HA installations

**Comment by kalmarzs** [comment]
Almost the same XML can be used for HANA and TREX installations. The only difference is that HANA requires one additional parameter the sapMDC, which decides whether the database must be installed in multi-tenant mode or not. As this parameter could not be applied to TREX, therefore should be left out completely.
In case of distributed and HA installations there is also one more parameter must be set: sapVirtHostname.

**Comment by dmacvicar** [comment]
This can be merged

**Comment by sknorr** [comment]
Hi @dmacvicar, sorry for the delay -- I have a added another commit. Can you take a look?

File: xml/s4s_installation_sap.xml [hunk 1]
```diff
@@ -961,4 +961,151 @@ SAP-NetWeaver-740-SR2-Installation-Export-CD-3-3</screen>
...
+# The /sapmnt path (unix only)
+NW_GetSidNoProfiles.sapmnt = /sapmnt
+
+# The SAP System ID of the system to install
+NW_GetSidNoProfiles.sid = ##sid##
+
+# Will this system be unicode system?
+NW_GetSidNoProfiles.unicode = true
+
+NW_SAPCrypto.SAPCryptoFile = /data/SAP_CDs/745-UKERNEL-SAP-Unicode-Kernel-745/DBINDEP/SAPEXE.SAR
```
Comment (sknorr) [inline xml/s4s_installation_sap.xml]: Is it wise to keep this path in the docs? -- I suspect that if you use the same version and copy the files with YaST, this path is always the same. However, that is quite a lot of assumptions -- should we use a replacable here?

Comment (dmacvicar) [inline xml/s4s_installation_sap.xml]: Well, I agree is very specific, on the other hand, this is an example and most people want something to copy paste and change. It is hard to figure out what goes in there without a concrete value.

Comment (sknorr) [inline xml/s4s_installation_sap.xml]: Okay.

**Comment by sknorr** [comment]
Thanks!

**Pull request merged**